README.md

```diff
@@ -8,9 +8,10 @@ Chain operators can use the Bridged USDC Standard for the OP Stack to get Bridge
 
 
 ## Contracts
-> :exclamation: `L1OpUSDCFactory.sol` has been deployed to `0x7dB8637A5fd20BbDab1176BdF49C943A96F2E9c6`.
- 
-_`L1OpUSDCFactory.sol`_ - Factory contract to deploy and setup the `L1OpUSDCBridgeAdapter` contract on L1. Precalculates the addresses of the L2 deployments and triggers their deployment, by sending a transaction to L2.
+_`L1OpUSDCFactory.sol`_ - Factory contract to deploy and setup the `L1OpUSDCBridgeAdapter` contract on L1. Precalculates the addresses of the L2 deployments and triggers their deployment, by sending a transaction to L2. It has been deployed to
```

Comment:
> :exclamation: `L1OpUSDCFactory.sol` has been deployed at the following addresses: 
> - Ethereum Mainnet: `0x7dB8637A5fd20BbDab1176BdF49C943A96F2E9c6`.
> - Ethereum Sepolia: `0xC0A9A6cf5d6873523da5B372e326e0668871d377`

Can we keep it in this format?

We can leave it out the base-sepolia deployment, this `L1OpUSDCFactory` is only supposed to be deployed on an L1, like Eth mainnet and Eth sepolia

Reply:
sorry missed this earlier!

Reply:
oh interesting - is there another set of contracts for getting native USDC for L3s? / why do we need another set of contracts

Reply:
couldn't find any information from Circle about native USDC for L3s....which maybe means the standard applies recursively? As long as a native USDC exists on the L2 then you should be able to send the same transactions to fit the native-USDC standard for L3s right? So it would work for e.g. base (which has native USDC) but not e.g. mode (which doesn't).

maybe I'm mistaken

Reply:
That sounds correct, mostly want to exclude the base address as this protocol was only intended for L1 -> L2 for OP-Chains so the L2 -> L3 flow is untested and therefore experimental, but hypothetically if the chain has Native USDC e.g. Base and the L3 is op stack compatible it should work

Reply:
okay in that case do you mind if I put the deployments in the docs and just put a warning that they're experimental? I'm testing it right now :)

Reply:
I've been deploying this on a ton of base-sepolia testnets and there are no issues!